avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -207,14 +213,15 @@ public Object convert(Schema schema, Object value) {
     });
   }
 
+  static final String AVRO_PROP = "avro";
   static final String AVRO_LOGICAL_TYPE_PROP = "logicalType";
   static final String AVRO_LOGICAL_TIMESTAMP_MILLIS = "timestamp-millis";
   static final String AVRO_LOGICAL_TIME_MILLIS = "time-millis";
   static final String AVRO_LOGICAL_DATE = "date";
   static final String AVRO_LOGICAL_DECIMAL = "decimal";
   static final String AVRO_LOGICAL_DECIMAL_SCALE_PROP = "scale";
   static final String AVRO_LOGICAL_DECIMAL_PRECISION_PROP = "precision";
-  static final String CONNECT_AVRO_DECIMAL_PRECISION_PROP = "avro.precision";
+  static final String CONNECT_AVRO_DECIMAL_PRECISION_PROP = "connect.decimal.precision";
```

Comment:
I believe the need here was that if you read in Avro data, the Connect decimal type doesn't include precision by default (it uses whatever precision is provided by the unscaled integer). To carry this across Avro -> Connect -> Avro conversions and still get the same Avro schema, you need to include this information.

With that in mind, perhaps the name doesn't matter since it's just used for the in-memory Connect representation. In theory changing it could be a problem if you mix and match converters, but in practice you're probably always using the same version of the converter when converting to Connect schemas and back again since this all happens in memory. So I think it'll be safe to change this, although the reason it was previously prefixed with `avro` is because it is actually an Avro-specific thing (since we don't include precision info in Connect Decimals).